src/components/APIDoc/SchemaPropertyView.tsx

```diff
@@ -88,6 +88,8 @@ export const ExtraPropertyView:React.FunctionComponent<ExtraPropertyViewProps> =
   return(
     <LabelGroup>
       {propSchema.format && <Label isCompact>{propSchema.format}</Label>}
+      {propSchema.description && <Level><Text>{propSchema.description}</Text></Level>}
+      {propSchema.example && <LabelGroup categoryName="Example"><Label>{propSchema.example}</Label></LabelGroup>}
```

Comment:
Could you share what this looks like? A screenshot is always helpful in PRs that update the UI

Reply:
![Screenshot from 2023-09-06 12-15-22](https://github.com/RedHatInsights/api-documentation-frontend/assets/111240801/2c0c98f9-4cc5-4a74-8496-9245f79d7c25)

Reply:
I didn't know you could add pictures on here! I added it on the story as well :)

Reply:
I'm not sure about the example :thinking: .

Maybe adding it as global example for the top level type instead of on each attribute? Like we do in the responses:

![image](https://github.com/RedHatInsights/api-documentation-frontend/assets/3845764/0e874994-5162-4e08-99da-ef1e4bb7d7dc)


@tahmidefaz WDYT?

Reply:
I think that makes sense to me. The example tag is a bit awkward :thinking:

Reply:
sounds good, will update